src/content/doc-surrealql/clauses/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 1
+sidebar_label: Clauses
+title: Clauses | SurrealQL
+description: Clauses are used to 
+---
+
+# Clauses
+
+In SurrealQL, clauses can be used to alter the way a query is executed. They are used in the following ways: 
+
+- [`EXPLAIN`](/docs/surrealql/clauses/explain): Explain the query plan.
+- [`FETCH`](/docs/surrealql/clauses/fetch): Fetch related records.
+- [`FROM`](/docs/surrealql/clauses/from): Specify the table or view to query.
```

Comment:
Could be really general plus hint at multiple targets with "Specify the target(s) to query from" or "Specify the target or targets to query from".

Or maybe "Specify the table(s) or other target(s) to query from" though that looks a bit uglier.